BurnOutSharp/PackerType/InnoSetup.cs

```diff
@@ -18,6 +19,12 @@ public class InnoSetup : IContentCheck, IScannable
                 new ContentMatch(new byte?[] { 0x49, 0x6E, 0x6E, 0x6F }, start: 0x30, end: 0x31),
                 GetVersion,
                 "Inno Setup"),
+
+            new ContentMatchSet(new byte?[]
```

Comment:
Since this is a notably different way of checking, you should probably create a new `GetVersion` method for this. Also, can you have the comment above this check with the string that this represents?